app/models/setting.rb

```diff
@@ -0,0 +1,17 @@
+# RailsSettings Model
+class Setting < RailsSettings::Base
+  cache_prefix { "v1" }
+
+  # Define your fields
+  # field :host, type: :string, default: "http://localhost:3000"
+  # field :default_locale, default: "en", type: :string
+  # field :confirmable_enable, default: "0", type: :boolean
+  # field :admin_emails, default: "admin@rubyonrails.org", type: :array
+  # field :omniauth_google_client_id, default: (ENV["OMNIAUTH_GOOGLE_CLIENT_ID"] || ""), type: :string, readonly: true
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is too long. [118/100]](https://app.codacy.com/manual/VoxPupuli/vox-pupuli-tasks/pullRequest?prid=5443028)